test/domain/post/getByFollowingAggregated.spec.ts

```diff
@@ -0,0 +1,26 @@
+
+import getByFollowingAggregated from '^/domain/post/getByFollowingAggregated/feature';
+import { beforeEach, describe, expect, it } from 'vitest';
+import { DATABASES, FILE_STORES, REQUESTERS } from './fixtures';
+
+beforeEach(async () =>
+{
+    await Promise.all([
+
+        DATABASES.withCreatorsPostsAndRelations(),
+        FILE_STORES.withImage()
+
+    ]);
+});
+
+describe('domain/post/getByFollowingAggregated', () =>
+{
+    it('should get  all posts from creators following the requester', async () =>
```

Comment:
This description is bound to the requirements, but I don't think it is correct... Besides that, three are spaces between 'get' and 'all'.